libnavigation-core/api/current.txt

```diff
@@ -397,6 +397,13 @@ package com.mapbox.navigation.core.routeoptions {
 
 }
 
+package com.mapbox.navigation.core.routerefresh {
+
+  public final class RouteRefreshControllerKt {
```

Comment:
I'm not sure how `metalava` passed it 🤔  RouteRefreshController is still internal

Reply:
I think the file-level constant was causing an issue for metalava. Gone with `companion object`.